### PR TITLE
73444 download attachment from action

### PR DESCRIPTION
**Pull request by llykkelarsen**
Endpoint for download attachment from an action
Tests

### CODE REVIEW

**Comment by llykkelarsen** [comment]
@janid1967 Did we agree to rename from _get_ to _download_ on the get*attachment(s) endpoints?

**Comment by janid1967** [comment]
Please add 2 tests in AccessValidatorTests.cs You find what I mean if you see GetTagAttachmentsQuery

**Comment by janid1967** [comment]
> @janid1967 Did we agree to rename from _get_ to _download_ on the get*attachment(s) endpoints?

nahh ... since we also use endpoint to get just url I think it's fine as it is

File: src/tests/Equinor.Procosys.Preservation.Query.Tests/GetActionAttachment/GetActionAttachmentQueryHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+using Equinor.Procosys.Preservation.Query.GetActionAttachment;
+using Equinor.Procosys.Preservation.Test.Common;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using ServiceResult;
+using Action = Equinor.Procosys.Preservation.Domain.AggregateModels.ProjectAggregate.Action;
+
+namespace Equinor.Procosys.Preservation.Query.Tests.GetTagAttachment
```
Comment (janid1967) [inline src/tests/Equinor.Procosys.Preservation.Query.Tests/GetActionAttachment/GetActionAttachmentQueryHandlerTests.cs:17]: wrong ns

**Pull request merged**